web/api/views.py

```
@@ -14,7 +14,7 @@
 from rest_framework import viewsets, status
 from rest_framework.response import Response
 from rest_framework.views import APIView
-from rest_framework.status import HTTP_400_BAD_REQUEST, HTTP_204_NO_CONTENT
+from rest_framework.status import HTTP_400_BAD_REQUEST, HTTP_204_NO_CONTENT, HTTP_202_ACCEPTED
```

Ruff

`rest_framework.status.HTTP_202_ACCEPTED` imported but unused

Remove unused import: `rest_framework.status.HTTP_202_ACCEPTED`

(F401)